cnudie/access.py

```diff
@@ -22,3 +26,94 @@ def s3_access_as_blob_descriptor(
         size=size,
         name=name or f's3://{s3_access.bucketName}/{s3_access.objectKey}',
     )
+
+
+def create_resolve_access(
```

Comment:
please mv to using code (-> delivery-service)

Reply:
Is required in cc-utils for BDBA cli